### ISSUE TITLE
About delly filter -s samples.tsv

### ISSUE THREAD

**Issue by ClarenceHsiang** [dellytools/delly#295]
![image](https://user-images.githubusercontent.com/57841470/182142109-136cfe5f-518b-4ec6-9aaa-4f92ea826338.png)
<img width="947" alt="NK_HYNK_vcf" src="https://user-images.githubusercontent.com/57841470/182142182-4782028d-32e3-4b95-b264-23dcf1267fbb.png">

Dear delly team:

I used conda to run delly, and I face a problem in filter:
The command I used as below:
>delly filter -f somatic -o NK_HYNK.pre.bcf -s NK_HYNK_sample.tsv NK_HYNK.bcf

Then, it showed this error 
![image](https://user-images.githubusercontent.com/57841470/182142478-8edd573a-a276-4a39-8a58-3687cf04461a.png)

May you give me some suggestions?  Thank you

**Comment by tobiasrausch** [comment]
Looks like some problem with the sample sheet. Can you please do:

`dos2unix NK_HYNK_sample.tsv`

Afterwards check that the columns are okay, e.g., 

`cut -f 1 NK_HYNK_sample.tsv`

and 

`cut -f 2 NK_HYNK_sample.tsv`

should give you the sample names and tumor or control.

**Comment by acarbome7** [comment]
Hello!

I am facing a similar problem... When I run:

`delly filter -f somatic -o "$SV_outDir"DELLY/012_pre.bcf -s "$input_Dir"delly_sampleDescriptionFile.txt "$SV_outDir"DELLY/012.bcf`

I get: "No tumor samples specified."

This is how the delly_sampleDescriptionFile.txt looks like: 
012-0034-01ND	control 
012-02-1TD	tumor 

I made:
`dos2unix delly_sampleDescriptionFile.txt`
but does not work. Then I made:

`cut -f 1 delly_sampleDescriptionFile.txt`
012-0034-01ND
012-02-1TD

`cut -f 2 delly_sampleDescriptionFile.txt`
control
tumor

I cannot figure out what's wrong... Any other suggestions? 
Thank you so much for your time

**Comment by tobiasrausch** [comment]
Maybe the sample names do not match the input BCF file? Can you try:

`bcftools view "$SV_outDir"DELLY/012.bcf | grep -m 1 "^#CHROM"`

**Comment by NidhiAgrawal27** [comment]
Hi. Were you able to find a solution to this?

**Comment by acarbome7** [comment]
yes, I did because Delly works very well for me, but I cannot recall what was wrong. What's your question?

**Comment by NidhiAgrawal27** [comment]
Hey!

Thank you so much for responding. I solved it. In my case spelling of 'tumor' in samples.tsv was the issue.  I was using 'tumour'.

Best 
Nidhi

**Comment by vaibhakv** [comment]
I have the same problem. I have spelled tumor and normal correctly. What else might be the issue?
